agenta-web/package.json

```diff
@@ -91,6 +92,6 @@
         "@types/node": "^20.8.10",
         "cypress": "^13.4.0",
         "node-mocks-http": "^1.12.2",
-        "prettier": "^3.0.0"
+        "prettier": "^3.2.5"
```

Comment:
right but the workflow kept failing because other dependency is using the newer verison.
The good thing is that with the new version there is no one single line reformatted so the end goal is still the same.